tests/test_registration.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
+import json
+import pytest
+from uuid import uuid4
+from random import randint
+from faker import Faker
+from api.services.registration_service import *
+from api.models import *
+
+faker = Faker()
+
+# Load the json payload
+payload = json.load(open("tests/payloads/registration_payload.json"))
+
+# Generate a unique flow_run_uuid
+new_run_uuid = str(uuid4())
+
+# Insert into the payload
+payload["flow_run_details"]["uuid"] = new_run_uuid
+
+
+def pytest_namespace():
+    # We will record the content version from the create operation for future reference
+    return {"initial_content_version_id": None}
+
+
+def test_create_call_log(app, db, setup_test_environment):
```

Comment:
Can we rename this function, doesn't seem like the test case function is for the call log creation.